ADR/ADR025-use-aasm-to-manage-form-state.md

```diff
@@ -0,0 +1,75 @@
+# ADR025: Use AASM to manage form state 
+
+Date: 2024-02-05
+
+## Status
+
+> Accepted
+
+## Context
+
+### Before
+
+Initially, there only three "states" that a form could be in, "Draft", "Live" and a combination of the two "Live with draft". These states were not hardcoded and we inferred by a combination of attribute  
+
+1. Draft - All "Form" records are draft
```

Comment:
```suggestion
1. Draft - All "Form" records start as in draft
```

Reply:
I think I've made this less clear 🤦

Reply:
What I was trying to say is that when looking at `Form` record before AASM was introduced, we should consider `Form` as being a draft record which a user can also edit at any point but it wouldn't be made live until a snapshot of it has been copied over to made_live_forms